tools/launcher/common/specdec/quantize_drafter.sh

```
@@ -0,0 +1,43 @@
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source ${SCRIPT_DIR}/../service_utils.sh
+
+trap 'error_handler $0 $LINENO' ERR
+
+###################################################################################################
+
+DRAFTER=${DRAFTER_CKPT}
+# Training writes exported-checkpoint-<step>/ under output_dir; take the newest.
+if [ ! -f "${DRAFTER}/config.json" ]; then
+    DRAFTER=$(ls -d ${DRAFTER_CKPT}/exported-checkpoint-* 2>/dev/null | sort -t- -k3 -n | tail -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)